unit-testing-react-what-why-how/README.md

```diff
@@ -1,13 +1,13 @@
-| Button | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/Button/problem) | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/Button/solution) |
-| Simplify-rock-paper-scissors | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/Simplify-rock-paper-scissors/problem) | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/Simplify-rock-paper-scissors/solution) |
-| test-rock-paper-scissors | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/test-rock-paper-scissors/problem) | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/test-rock-paper-scissors/solution) |
-| renderHook | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/renderHook/problem) | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/renderHook/solution) |
-| Temp - vehicle selector | [CodeSandbox](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/vehicle-selector-react) | Runs, but can't run tests... use for multiple exercises? |
+| Exercise                     | Problem                                                                                                                                                       | Solution                                                                                                                                                        |
+| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| getStringLength              | [Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/getStringLength)                                |                                                                                                                                                                 |
+| Basics-together              | [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/Basics-together/problem?file=/src/index.test.ts) | [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/unit-testing-react-what-why-how/Basics-together/solution?file=/src/index.test.ts) |
```

Comment:
This points to `index.test.ts` but it doesn’t work because the file has a `js` extension. I think it’d be good to use `ts`/`tsx` everywhere, so if someone wants to write TypeScript, they don’t have to fiddle with changing the file extension.

Reply:
Great catch! Wholeheartedly agree. I'm trying to swap them all out for `ts`/`tsx` files, so I'll add a commit updating the file.